contracts/fast/FastTokenFacet.sol

```diff
@@ -245,6 +246,14 @@ contract FastTokenFacet is AFastFacet, IERC20, IERC1404 {
     );
   }
 
+  /// IERC165 implementation.
```

Comment:
I disagree with the architecture here - but maybe I'm missing something. Let's touch base @jamesduncombe when we speak today.